cardano-cli/test/cardano-cli-golden/files/golden/help.cli

```diff
@@ -6800,8 +6800,13 @@ Usage: cardano-cli conway governance drep update-certificate
 
   Create a DRep update certificate.
 
-Usage: cardano-cli conway governance drep metadata-hash --drep-metadata-file FILEPATH
-                                                          [--out-file FILEPATH]
+Usage: cardano-cli conway governance drep metadata-hash 
+                                                          ( --drep-metadata-file FILEPATH
+                                                          | --drep-metadata-url TEXT
+                                                          )
+                                                          [ --expected-hash HASH
+                                                          | --out-file FILEPATH
+                                                          ]
 
   Calculate the hash of a metadata file.
```

Comment:
```suggestion
  Calculate the hash of a metadata file, optionally checking the obtained hash against an expected value.
```

Optional (maybe it dilutes the primary purpose of the command?)

Reply:
No, that is accurate. I didn't think to update this description